content/en/news/2019/announcing-1.3.5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.3.5
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-11-06
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.5
+aliases:
+    - /news/announcing-1.3.5
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.4 and Istio 1.3.5.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** Envoy listener configuration for TCP headless services. ([Issue #17748](https://github.com/istio/istio/issues/17748))
+- **Fixed** an issue which caused stale endpoints to remain even when a deployment was scaled to 0 replicas. ([Issue #14436](https://github.com/istio/istio/issues/14336))
+- **Fixed** Pilot to gracefully handle bad configurations for invalid custom Envoy filters. ([Issue 17266](https://github.com/istio/istio/issues/17266))
```

Comment:
This is not fully accurate -- it is not just EnvoyFilters, but handles a common class of Pilot bugs gracefully.

Also `Envoy filter` is misleading, and this should probably be EnvoyFilter (poorly named :slightly_smiling_face: )